components/raftstore-v2/src/batch/apply.rs

```diff
@@ -50,15 +50,17 @@ pub struct ApplyPoller {
 
 impl ApplyPoller {
     pub fn new(apply_ctx: ApplyContext, cfg_tracker: Tracker<Config>) -> ApplyPoller {
-        ApplyPoller {
+        let mut poller = ApplyPoller {
             apply_task_buf: Vec::new(),
             pending_latency_inspect: Vec::new(),
             apply_ctx,
             cfg_tracker,
-        }
+        };
+        poller.apply_buf_capacity();
```

Comment:
I don't think it's necessary, it's always applied in `begin`.

Reply:
Okay, I just copied it from store poller.